vars/main.yml

```
@@ -49,7 +49,7 @@ vip_manager_mask: "24"  # netmask for the virtual ip
 
 # DCS (Distributed Consensus Store)
 dcs_exists: false  # or 'true' if you don't want to deploy a new etcd cluster
-dcs_type: "etcd"  # or 'consul'
+dcs_type: "patroni_raft"  # or 'consul' or 'etcd'
```

```suggestion
dcs_type: "etcd"  # or 'consul' or 'patroni_raft'
```

patroni_raft is currently in the beta stage. 

I suggest leaving the default configuration using etcd

besides:

https://github.com/zalando/patroni/blob/master/docs/releases.rst#version-300
> The RAFT support is deprecated. We will do our best to maintain it, but take neither guarantee nor responsibility for possible issues.

---

apply before merging (after tests)